components/event-handler-notification/org.wso2.carbon.identity.event.handler.notification/src/main/java/org/wso2/carbon/identity/event/handler/notification/NotificationConstants.java

```diff
@@ -18,13 +18,23 @@
 
 package org.wso2.carbon.identity.event.handler.notification;
 
+import org.wso2.carbon.identity.core.util.IdentityUtil;
+
 public class NotificationConstants {
 
     private NotificationConstants() {
     }
 
+    private static String getClaimUriLocale() {
```

Comment:
add new line after method signature

Reply:
Fixed in commit c6f1d06820885092027e0f28adc4ac8b890cce3e